startup/setup_aws.rb

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env ruby
+# Only allow this to run in development
+return unless ENV["EXERCISM_ENV"]=="development"
+
+require "bundler/setup"
+require "exercism_config"
+
+puts "Create AWS/DynamoDB tables..."
+
+Exercism.config.dynamodb_client = ExercismConfig::SetupDynamoDBClient.()
+
+##################
+# Setup local s3 #
+##################
+
+# We don't need this running for CI atm as none of our
+# tests actually hit s3. Although we might choose to change this
+unless ENV["EXERCISM_CI"]
+  begin
+    ExercismConfig::SetupS3Client.().create_bucket(bucket: Exercism.config.aws_iterations_bucket)
+  rescue Seahorse::Client::NetworkingError => e
+    puts "local S3 not up yet..."
+    sleep 2 # slighty retry delaty
+    retry
```

Comment:
Should we have a max number of retries or do we not care? I guess you'll notice anyway if it just keeps replaying that?

Reply:
Yeah, since this is only for local dev that was my thought that it'll either sort itself out or be noticed and being loud is a good thing for debugging.

Reply:
Sounds good.